DependencyInjection/Configuration.php

```diff
@@ -186,7 +186,6 @@ private function addViewSection(ArrayNodeDefinition $rootNode)
                             ->canBeUnset()
                             ->children()
                                 ->scalarNode('callback_param')->defaultValue('callback')->end()
-                                ->scalarNode('callback_filter')->defaultValue('/(^[a-z0-9_]+$)|(^YUI\.Env\.JSONP\._[0-9]+$)/i')->end()
```

Comment:
you should first add a deprecation warning in 1.x versions when configuring this, to warn people *before* they upgrade (Symfony does not log deprecations triggered that early for now, but there is a pending PR for that)